packages/adminrouter/extra/src/includes/http/common.conf

```diff
@@ -1,3 +1,6 @@
+# Enable VTS module for metrics
+vhost_traffic_status_zone shared:vhost_traffic_status:32m;
```

Comment:
This setting is for the size of the shared memory region that the metrics counters are written to.
When using dynamic metric generation e.g. based on URIs accessed the data in there can grow arbitrarily large. Currently the VTS module does not have a cleanup mechanism. So the way to prevent this is to use a static set of metrics. I've also set the region precautiously to 32MB instead of 8MB.